### ISSUE TITLE
Translation memory not working in Matecat

### ISSUE THREAD

**Issue by MariusJulius** [keeleinstituut/tv-tolkevarav#811]

**Comment by MariusJulius** [comment]
As @NeleKo tested again it worked but is slow in general, can it be due to low specs on test system? @kadmit

**Comment by kadmit** [comment]
@MariusJulius Yes, that's the exact reason. But for the test environment, it's fine as we use it only for testing purposes. 


Regarding production it will work the same way as the test rn, so I think we need to communicate with the hosting provider and get the info on how much it would cost the client to increase the RAM, up to 8GB.
